content/en/platform/corda/4.8/open-source/key-concepts-notaries.md

```diff
@@ -21,74 +21,42 @@ title: Notaries
 ## Summary
 
 * *Notary clusters prevent “double-spends”*
-* *Notary clusters are also time-stamping authorities. If a transaction includes a time-window, it can only be notarised during that window*
+* *Notary clusters are also time-stamping authorities. If a transaction includes a time-window, it can only be notarized during that window*
 * *Notary clusters may optionally also validate transactions, in which case they are called “validating” notaries, as opposed to “non-validating”*
-* *A network can have several notary clusters, each running a different consensus algorithm*
+* *A network can have several notary clusters*
 
 ## Video
 
 {{% vimeo 214138458 %}}
 
 ## Overview
 
-A *notary cluster* is a network service that provides **uniqueness consensus** by attesting that, for a given
-transaction, it has not already signed other transactions that consumes any of the proposed transaction’s input states.
+The notary cluster is Corda's uniqueness consensus service. The notary's role is to ensure a
```

Comment:
I'd start with a simpler explanation that notaries prevent double spends (and define double spends). 

I also think it's worthwhile to a) link to the consensus key concept doc and b) briefly note that this is a concept unique to corda, because most blockchains reach consensus through broadcasting/byzantine fault tolerance. (i.e. we are not burning the planet!) without going too deep into consensus as it's a different topic.